### PR TITLE
feat: hide first default story in styles [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Description:
Similar to the behaviour of the components will hide the "default" story preventing to have it duplicated to styles.

## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

**Comment by mariohamann** [comment]
Why should this be hidden? The `styles` are meant to be interactive.

**Review by mariohamann — APPROVED** [review]

File: packages/components/.storybook/preview-head.html [hunk 1]
```diff
@@ -140,4 +140,8 @@
   h2#stories + [id^='anchor--components-'].sb-anchor {
```
Comment (smfonseca) [inline packages/components/.storybook/preview-head.html:140]: You can add the selectors after the first one so:
`h2#stories + [id^='anchor--components-'].sb-anchor, h2#stories + [id^='anchor--styles-'].sb-anchor`

File: packages/components/.storybook/preview-head.html [hunk 2]
```diff
@@ -140,4 +140,8 @@
...
     display: none;
   }
+
+  h2#stories + [id^='anchor--styles-'].sb-anchor {
+    display: none;
+  }
 </style>
```
Comment (mariohamann) [inline packages/components/.storybook/preview-head.html:147]:
```suggestion
  h2#stories + [id^='anchor--components-'].sb-anchor,
  h2#stories + [id^='anchor--styles-'].sb-anchor {
    display: none;
  }
```

**Comment by mariohamann** [comment]
Pls fix linting/formatting:

**Review by yoezlem — APPROVED** [review]

**Pull request merged**